airflow/www/templates/airflow/dags.html

```diff
@@ -309,6 +307,21 @@ <h2>DAGs</h2>
           }
         });
       });
+      d3.json("{{ url_for('Airflow.last_dagruns') }}", function(error, json) {
+        for(var safe_dag_id in json) {
+          dag_id = json[safe_dag_id].dag_id;
+          last_run = json[safe_dag_id].last_run;
+          g = d3.select('div#last-run-' + safe_dag_id)
+          g.selectAll('a')
+            .attr("href", "{{ url_for('Airflow.graph') }}?dag_id=" + dag_id + "&execution_date=" + last_run)
```

Comment:
```suggestion
            .attr("href", "{{ url_for('Airflow.graph') }}?dag_id=" + encodeUriComponent(dag_id) + "&execution_date=" + last_run)
```

Reply:
Can make this change, seems safe enough and good in general. Note not encoding dag id is a common pattern in the UI and probably needs to be factored into some method.